From 79ba88e0aad24ae1bb07c39285fed65ba754a9f7 Mon Sep 17 00:00:00 2001
From: Nathan Mittler <nmittler@gmail.com>
Date: Mon, 16 May 2022 14:49:58 -0700
Subject: [tf] Add more utils to echo containers (#38947)

This is in part to force a rebuild of the base images, now that the regex has been expanded to detect changes to the echo base images (https://github.com/istio/test-infra/pull/4047).
---
 pkg/test/echo/docker/Dockerfile.app_sidecar_base        | 3 +++
 pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/pkg/test/echo/docker/Dockerfile.app_sidecar_base b/pkg/test/echo/docker/Dockerfile.app_sidecar_base
index 84bbf61546..cb733b13cc 100644
--- a/pkg/test/echo/docker/Dockerfile.app_sidecar_base
+++ b/pkg/test/echo/docker/Dockerfile.app_sidecar_base
@@ -15,6 +15,9 @@ RUN apt-get update && \
     iproute2 \
     sudo \
     curl \
+    netcat \
+    tcpdump \
+    conntrack \
     net-tools \
     ca-certificates \
     && apt-get upgrade -y \
diff --git a/pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos b/pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos
index 1b0db5bb0d..01a01b39fc 100644
--- a/pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos
+++ b/pkg/test/echo/docker/Dockerfile.app_sidecar_base_centos
@@ -8,6 +8,9 @@ RUN yum install -y \
     iproute \
     sudo \
     curl \
+    netcat \
+    tcpdump \
+    conntrack \
     net-tools \
     ca-certificates \
     && update-ca-trust \
-- 
2.35.3

